turtlebot2: #namespace
    task_and_robot_environment_name: 'MyTurtleBot2Maze-v0'
    ros_ws_abspath: "/home/eldar/python3_ws"
    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command
    
    #qlearn parameters
    alpha: 1.0
    gamma: 0.7
    epsilon: 0.9
    epsilon_discount: 0.999
    nepisodes: 400
    nsteps: 400
    running_step: 0.06 # Time for each step

